pages/api/trigger-feature.js

```diff
@@ -0,0 +1,22 @@
+import { Event, UserIdentified, AccountIdentified } from "@journyio/sdk";
+import { client } from "../../util/journy";
+import getSession from "../../util/getSession";
+
+async function handler(request, response) {
+  const { featureName, accountId } = request.body;
+  const user = request.session.get("user");
+
+  await client.addEvent(
+    Event.forUserInAccount(
+      "trigger-feature",
```

Comment:
```suggestion
      featureName,
```

Let's set the feature name as event name so that we can use them as different events?